src/views/ProfileView.vue

```diff
@@ -1,18 +1,24 @@
 <template>
   <div class="page-container">
-    <div class="page-title">{{ $t('profile.myProfile')}}</div>
+    <div class="page-title my-0">
+      {{ $t('profile.myProfile')}}
+    </div>
     <div class="grid">
-      <div class="col-12 md:col-6">
+      <div class="col-6 md:col-6">
+        <UserInfoComponent :user="current.user as UserResponse"/>
+      </div>
+      <div class="col-6 md:col-6">
           <ChangePinComponent />
       </div>
-      <div class="col-12 md:col-6">
+      <div class="col-6 md:col-6">
         <ChangePasswordComponent />
       </div>
-      <div class="col-12 md:col-6">
+      <div class="col-6 md:col-6" v-if="isAdmin">
         <ChangeApiKeyComponent />
-      </div>
-      <div class="col-12 md:col-6">
-        <ChangeInfoComponent />
+        <div class="flex flex-row mb-2 align-items-center">
+          <h3 class="mr-3">{{ $t('profile.individualDataAnalysis') }}</h3>
+          <InputSwitch v-model="dataAnalysis" @update:modelValue="handleChange" />
```

Comment:
Why is this only allowed when you are an admin?

Reply:
fixed!